.github/workflows/gradle-build-poc.yml

```diff
@@ -0,0 +1,134 @@
+name: Gradle Build & Deploy - POC
+on:
+  push:
+    branches:
+      # - 'develop'
+      - 'feature/2486-create-new-pipeline-for-updated-build-deployment'
+env:
+  PROJECT_ID: ${{ secrets.PROJECT_ID }}
+  PROJECT_NAME: ${{ secrets.PROJECT_NAME }}
+  RUN_REGION: us-central1
+  SERVICE_NAME: checkins-develop
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment:
+      name: DEV-POC
+      # url: ${{ env.CLOUD_RUN_ADDRESS }}/
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - id: 'auth_with_gcp'
+        uses: 'google-github-actions/auth@v2'
+        with:
+          project_id: '${{env.PROJECT_ID}}'
+          workload_identity_provider: '${{secrets.WORKLOAD_IDENTITY_PROVIDER}}'  
+          service_account: '${{secrets.GCLOUD_SERVICEACCOUNT}}'
```

Comment:
Thoughts on naming this secret something like `AUTOMATION_SA_KEY` so we know its purpose at a glance?

Reply:
I would compromise on AUTOMATION_SA.  The value here is really more of an ID similar to this: github-wif@checkins-dev-425419.iam.gserviceaccount.com.  When I see KEY, I think of something different.